.ci/magic-modules/create-pr.sh

```diff
@@ -63,7 +63,7 @@ if [ "$BRANCH_NAME" = "$ORIGINAL_PR_BRANCH" ]; then
     fi
 
     git checkout -b "$BRANCH_NAME"
-    if ANSIBLE_PR=$(hub pull-request -b "$ANSIBLE_REPO_USER/ansible:devel" -F ./downstream_body); then
+    if ANSIBLE_PR=$(hub pull-request -b "$ANSIBLE_REPO_USER/ansible:rebase" -F ./downstream_body); then
```

Comment:
I feel suspicious about this.  Is this the plan as discussed?  Can you elaborate on why this is necessary?

Reply:
I was trying to create a PR that was heading to the `rebase` branch.

For whatever reason, the PR for Ansible I have right now looks how I want it to. It adds a whole bunch of commits to devel, but keeps the commit history in tact. There's a merge commit (I made that commit myself) and then the difference between the merge commit + HEAD is the difference I would expect from this PR.

Reply:
Truth be told, I haven't thought about this in quite a while and I'm not entirely sure what the plan we agreed upon was.

Reply:
Undo this stuff and you're good!